test/gui/shared/scripts/helpers/api/HttpHelper.py

```diff
@@ -28,3 +28,7 @@ def delete(url, headers={}, user=None):
 
 def mkcol(url, headers={}, user=None):
     return send_request(url=url, method="MKCOL", headers=headers, user=user)
+
+
+def propfind(url, headers={}, user=None):
```

Comment:
PROPFIND can have request data
```suggestion
def propfind(url, body=None, headers={}, user=None):
```